package.json

```diff
@@ -109,7 +109,7 @@
     "@babel/runtime": "7.23.8",
     "@chelonia/cli": "2.2.3",
     "@exact-realty/multipart-parser": "1.0.12",
-    "@exact-realty/rfc8188": "1.0.5",
+    "@apeleghq/rfc8188": "1.0.7",
```

Comment:
Reminder to bump `@sbp/okturtles.events`

Reply:
Not sure if it should be done is this PR though, as it has some breaking changes.

Reply:
I don't think those changes affect anything we're doing

Reply:
Sure, I can do it now, but you answered your own question (emphasis mine, of course):

> _I don't think_ those changes [...]

I need to go through every call to `/once` and see to it that we're using the API correctly.